i18n/de_DE/data/validation.json

```diff
@@ -0,0 +1,30 @@
+{
+	"address" : {
+		"salutation": "^(Herr|Frau)$",
+		"title": "",
+		"companyName": "^.+$",
+		"firstName": "^[A-Za-z\u00C0-\u00D6\u00D8-\u00f6\u00f8-\u00ff\\s\\-\\.\\']+$",
+		"lastName": "^[A-Za-z\u00C0-\u00D6\u00D8-\u00f6\u00f8-\u00ff\\s\\-\\.\\']+$",
+		"street": "^.+$",
+		"city": "^.+$",
+		"postcode": "^.+$",
+		"country": "^.+$",
+		"email": "^(.+)@(.+)\\.(.+)$"
+	},
+	"contactForm": {
+		"firstName": "^.+$",
+		"lastName": "^.+$",
+		"donationNumber": "^[0-9]*$",
+		"email": "^(.+)@(.+)\\.(.+)$",
+		"topic": "^.+$",
+		"subject": "^.+$",
+		"comment": "(\n|.)+"
+	},
+	"dateOfBirth": [
```

Comment:
This needs some documentation:

1) How is this array format used? Are those concatenated parts (concatenated with `|`)?
2) What's the pattern here, expressed in plain English? I get the month name check (possibly also with a leap year check), but what are there edge cases in the leap year check? What does the year check do (upper and lower bound)? 
3) What's the expected format (separator, year-month-day order) here?
4) It's totally fancy, but is really necessary? Couldn't the same result be achieved with platform date checks? I know JavaScript date handling/validation is bad, but is worse than this monster regex?